src/cbor.h

```diff
@@ -390,8 +390,9 @@ struct CborParser
 {
     union {
         const uint8_t *end;
-        const struct CborParserOperations *ops;
-    } source;
+        void *ctx;
```

Comment:
Should we add a `cbor_parser_get_user_context()` so the user doesn't need to know about the internals of the struct? Then we don't have to document it either.

Reply:
Sounds fair, I'll add some "standard" getters/setters for structure hiding purposes.

Reply:
Like I said, we can do this after we merge your change.